### ISSUE TITLE
missing final directory

### ISSUE THREAD

**Issue by S-Stephen** [TobyBoyne/gf2-software#1]
Is this the correct repository for P3 - Team19? 

To be sure that we are running the correct version from the repository we were expecting a 'final' directory containing the deploy-able code.

**Comment by TobyBoyne** [comment]
Yes this is the correct repository - we will make and push this directory shortly.

**Comment by S-Stephen** [comment]
Thank you - please don't forget the test definition files also

**Comment by TobyBoyne** [comment]
The final folder has been added. Test definition files have been given in the `tests/test_files` folder (the path to this directory has been added to the README for reference).
